workflow/rules/eval.smk

```diff
@@ -13,9 +13,22 @@ rule rename_contigs:
         "-Ob -o {output} 2> {log}"
 
 
-use rule normalize_truth as normalize_calls with:
+rule remove_non_pass:
     input:
         get_callset,
+    output:
+        "results/filtered-variants/{callset}.bcf",
```

Comment:
Did you use `.bcf` on purpose here? We found that we had to add a converting line to the vcf-genotype-annotator because it can only handle `.vcf` and `.vcf.gz`.  We also changed that in rule `rename_contigs` (see #63).

Reply:
Well, if the steps can handle bcf, that is always preferable because it is much smaller and more efficient to read and write. If there is a step that cannot handle bcf, one can either use something like `<(bcftools view {input})`, or write to vcf.gz before that step. But here, since the step using this output is able to handle bcf, there is no need to do that.